pom.xml

```diff
@@ -52,6 +52,7 @@
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
         <commons.codec.version>1.15</commons.codec.version>
         <snakeyaml.version>1.28</snakeyaml.version>
+        <jackson.version>2.16.0</jackson.version>
```

Comment:
As discussed over the call, let's make this change in the parent pom only. That would facilitate others to not pin their individual versions to fix this CVE, rather just upgrade the parent pom version.